src/redux/reducers/profileReducer.js

```diff
@@ -1,11 +1,18 @@
 import {
   PROFILE_REQUEST, PROFILE_SUCCESS, PROFILE_FAILURE,
   EDIT_PROFILE_FAILURE, EDIT_PROFILE_REQUEST, EDIT_PROFILE_SUCCESS,
+  FOLLOW_FAILURE, FOLLOW_REQUEST, FOLLOW_SUCCESS,
+  UNFOLLOW_FAILURE, UNFOLLOW_REQUEST, UNFOLLOW_SUCCESS
 } from '../actiontypes';
+import { CommentActions } from 'semantic-ui-react';
 
 const initialState = {
   isFetching: false,
-  profile: {}
+  profile: {},
+  followed: null,
+  followers_count: null,
```

Comment:
use `carmel case` for your variable names

Reply:
Fixed it